src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,98 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.stream.IntStream;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_SIZE_ARRAY = 10;
+    private T[] array;
+    private int size = 0;
```

Comment:
```suggestion
    private T[] elements;
    private int size;
```

`size` variable will be initialized by 0 by default